### PR TITLE
[candi][openstack] Add warning in docs about availability zones for CloudPermanent nodes

### PR DESCRIPTION
**Pull request by unreturned**
## Description
Add warning in docs about availability zones for CloudPermanent nodes.

## Why do we need it, and what problem does it solve?
We have not correct calculation for zones for CloudPermanent nodes, that means if we have only one zone in `volumeTypeMap` by CloudPermanent NodeGroup like this:
```
nodeGroups:
- name: front
  replicas: 2
  nodeTemplate:
    labels:
      node-role.deckhouse.io/frontend: ""
    taints:
      - effect: NoExecute
        key: dedicated.deckhouse.io
        value: frontend
  instanceClass:
    flavorName: STD3-2-4
    imageName: ubuntu-22-202404160933.gitd6495fe9
    rootDiskSize: 30
    mainNetwork: kovalkov
  volumeTypeMap:
    ME1: high-iops
```
nodes will be created in different zones (all discovered) like this:
```
│ │ │   + resource "openstack_blockstorage_volume_v3" "volume" {
│ │ │       + availability_zone    = "GZ1"
│ │ │       + name                 = "kovalkov-front-0"
│ │ │   + resource "openstack_compute_instance_v2" "node" {
│ │ │       + availability_zone   = "GZ1"
│ │ │       + flavor_name         = "STD3-2-4"
│ │ │       + image_name          = "ubuntu-22-202404160933.gitd6495fe9"
│ │ │       + name                   = "kovalkov-front-0"
│ │ │   + resource "openstack_blockstorage_volume_v3" "volume" {
│ │ │       + availability_zone    = "ME1"
│ │ │       + name                 = "kovalkov-front-1"
│ │ │   + resource "openstack_compute_instance_v2" "node" {
│ │ │       + availability_zone   = "ME1"
│ │ │       + flavor_name         = "STD3-2-4"
│ │ │       + image_name          = "ubuntu-22-202404160933.gitd6495fe9"
│ │ │       + name                   = "kovalkov-front-1"
```
If we need limits by zone, we need to specify zones in `nodeGroups.zones` parametr, like this:
```
nodeGroups:
- name: front
  replicas: 2
  nodeTemplate:
    labels:
      node-role.deckhouse.io/frontend: ""
    taints:
      - effect: NoExecute
        key: dedicated.deckhouse.io
        value: frontend
  instanceClass:
    flavorName: STD3-2-4
    imageName: ubuntu-22-202404160933.gitd6495fe9
    rootDiskSize: 30
    mainNetwork: kovalkov
  volumeTypeMap:
    ME1: high-iops
  zones:
    - ME1
```
We need document this behaviour and gave information about workaround.

## Why do we need it in the patch release (if we do)?

Not necessarily.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: chore
summary: Add warning in docs about availability zones for CloudPermanent nodes.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @unreturned. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @unreturned. I've started the workflow for label 'deploy/web/test'...

File: ee/candi/cloud-providers/openstack/openapi/doc-ru-cluster_configuration.yaml [hunk 1]
```diff
@@ -192,9 +192,9 @@ apiVersions:
 
                 Формат элементов словаря: `<ЗОНА ДОСТУПНОСТИ>: <ТИП ДИСКА>` (см. пример).
 
-                Если значение, указанное в `replicas`, превышает количество элементов в словаре, узлы, чьи номера превышают
-                длину словаря, получают значения, начиная с начала словаря. Если для словаря из примера указано `replicas: 5`, с типом
-                диска `fast-eu-1a` будут worker-0, worker-2 и worker-4, а с типом диска `fast-eu-1b` будут worker-1 и worker-3.
+                Если значение, указанное в `replicas`, превышает количество элементов в словаре, узлы, чьи номера превышают длину словаря, получают значения, начиная с начала словаря. Если для словаря из примера указано `replicas: 5`, с типом диска `fast-eu-1a` будут worker-0, worker-2 и worker-4, а с типом диска `fast-eu-1b` будут worker-1 и worker-3.
+
+                > **Внимание!** Ограничение по зонам доступности работает только в связке с параметром `zones` (`nodeGroups.zones`). Если `zones` не задан, все зоны доступности будут использованы.
```
Comment (z9r5) [inline ee/candi/cloud-providers/openstack/openapi/doc-ru-cluster_configuration.yaml:197]:
```suggestion
                > **Внимание!** Ограничение по зонам доступности работает только при использовании параметра `zones` (`nodeGroups.zones`). Если параметр `zones` не задан, будут использованы все зоны доступности.
```

File: ee/candi/cloud-providers/openstack/openapi/cluster_configuration.yaml [hunk 2]
```diff
@@ -360,6 +360,8 @@ apiVersions:
 
                 If the value specified in `replicas` exceeds the number of elements in the dictionary, the nodes whose number exceeds the length of the dictionary get the values starting from the beginning of the dictionary. For example, if `replicas: 5`, then worker-0, worker-2, worker-4 will have the `fast-eu-1a` disk type, while worker-1, worker-3 will have the `fast-eu-1b` disk type.
 
+                > **Caution!** Limits by availability zones works only with specified parameter `zones` (`nodeGroups.zones`). If `zones` not specified, all availability zones will be used.
```
Comment (z9r5) [inline ee/candi/cloud-providers/openstack/openapi/cluster_configuration.yaml:363]:
```suggestion
                > **Caution!** Limits by availability zones works only when using the `zones` parameter (`nodeGroups.zones`). If `zones` parameter is not specified not specified, all availability zones will be used.
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10017](https://github.com/deckhouse/deckhouse/pull/10017) to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) successful!

**Pull request merged**